addon/components/paper-progress-linear.js

```diff
@@ -1,37 +1,38 @@
 import Ember from 'ember';
 import ColorMixin from 'ember-paper/mixins/color-mixin';
+const { inject, computed, Component, isPresent } = Ember;
+
 
 function makeTransform(value) {
-  var scale = value / 100;
-  var translateX = (value - 100) / 2;
+  let scale = value / 100;
+  let translateX = (value - 100) / 2;
   return 'translateX(' + translateX.toString() + '%) scale(' + scale.toString() + ', 1)';
 }
 
-const MODE_DETERMINATE = "determinate",
-  MODE_INDETERMINATE = "indeterminate",
-  MODE_BUFFER = "buffer",
-  MODE_QUERY = "query";
-
+const MODE_DETERMINATE = 'determinate',
+  MODE_INDETERMINATE = 'indeterminate',
+  MODE_BUFFER = 'buffer',
+  MODE_QUERY = 'query';
```

Comment:
make each const declaration separate. Similar to https://github.com/miguelcobain/ember-paper/blob/wip/v1.0.0/addon%2Fcomponents%2Fpaper-progress-circular.js#L8-L9

Again, sorry for this, but this will be in the jscs rules in a near future. So we're saving time upfront.